server.go

```diff
@@ -92,6 +93,14 @@ func NewFromConfig(conf Config) (ret Server, err error) {
 		}(ret.Workers[i])
 	}
 
+	ret.EventWorker = NewEventWorker(ret.statsd)
```

Comment:
The assumption being that we don't generate enough of these to warrant multiple workers, I assume?